### DISCUSSION TITLE
Clogged ears

### DISCUSSION THREAD

**Discussion by XelaAl** [NAIST-SE/AttractingContributors#329] [General]
How am I supposed to get rid of earwax? I tried a cold and it made things worse. Now my ears just feel clogged. What should I do?

**Comment by ellielo** [comment]
See a doctor as soon as possible

**Comment by hernessteve** [comment]
Just wait a few days and it will go away eventually. i always have clogged ears after the long flights and it starts to fade after one day normally.
